server/serializers.py

```diff
@@ -258,17 +505,58 @@ def jsonize_clientside_tab(tab: clientside.TabUpdate) -> Dict[str, Any]:
     return d
 
 
+def _localize_module_message(
+    message_id: str,
+    *,
+    default: Optional[str],
```

Comment:
Nix `default`. Where does it come from?

The answer is: "it comes when we're translating the specfile." But there's a better way to do that:

`def _localize_module_spec_message(message_path: str, ctx: JsonizeModuleContext, specfile_message: str)`

... and then simplify _this_ method by making it accept an `I18nMessage`.

Reply:
Although it's more meaningful when used for translating the specfile, the `default` argument also covers the case of `jsonize_i18n_message`, which has its own default value as per [this comment](https://github.com/CJWorkbench/cjworkbench/pull/174#discussion_r369620491).

Reply:
Please nix it anyway. The function itself has all the information it needs to generate it.